pkg/ebpfeng/ebpf_engine_interface.go

```diff
@@ -0,0 +1,7 @@
+package ebpfeng
+
+type EbpfEngineClient interface {
```

Comment:
[Question]: I'm not sure it is right to have the `Client` as part of an interface name